pkg/queue/prometheus_stats_reporter.go

```diff
@@ -114,10 +112,10 @@ func (r *PrometheusStatsReporter) Report(stat *autoscaler.Stat) error {
 		return errors.New("PrometheusStatsReporter is not initialized yet")
 	}
 
-	operationsPerSecondGV.With(r.labels).Set(stat.RequestCount)
-	proxiedOperationsPerSecondGV.With(r.labels).Set(stat.ProxiedRequestCount)
-	averageConcurrentRequestsGV.With(r.labels).Set(stat.AverageConcurrentRequests)
-	averageProxiedConcurrentRequestsGV.With(r.labels).Set(stat.AverageProxiedConcurrentRequests)
+	operationsPerSecondGV.With(r.labels).Set((stat.RequestCount / (r.reporterReportingPeriod.Seconds())))
```

Comment:
```suggestion
	operationsPerSecondGV.With(r.labels).Set(stat.RequestCount / r.reporterReportingPeriod.Seconds())
```

The parentheses are not needed here.

Reply:
Right, removed them

Reply:
Done